cypress/integration/project-error-screen.spec.ts

```diff
@@ -0,0 +1,46 @@
+import capitalize from "lodash/capitalize";
+import mockProjects from "../fixtures/projects.json";
+
+describe("Project Error Screen", () => {
```

Comment:
Great job implementing this test! Imo you could move this to the existing project page test as well. No need to have another test file I think